### ISSUE TITLE
CloudSequence.Parse

### ISSUE THREAD

**Issue by krontogiannis** [mbraceproject/MBrace.Core#19]
I think that `CloudSequence.Parse` (or any `.Parse`) should check that the given path exists.
```fsharp
let x = runtime.StoreClient.CloudSequence.Parse<int>(System.Guid.NewGuid().ToString())
val x : CloudSequence<int> = CloudSequence[System.Int32] at e13f61da-05dc-4028-b892-bb625a74313e
```
I think that delaying the error might be confusing.

**Comment by eiriktsarpalis** [comment]
Hmmm, I think I did this deliberately. If you run this with `force=true` it should do all the verifications there and then.

**Comment by eiriktsarpalis** [comment]
Fixed in f79a20dad58de884b78ba491330ea3a32e87acf1
